src/Microsoft.ML.Recommender/MatrixFactorizationPredictor.cs

```diff
@@ -375,13 +388,16 @@ public Row GetRow(Row input, Func<int, bool> active)
         }
     }
 
-    public sealed class MatrixFactorizationPredictionTransformer : PredictionTransformerBase<MatrixFactorizationPredictor>, ICanSaveModel
+    /// <summary>
+    /// Trains a <see cref="MatrixFactorizationModelParameters"/>. It factorizes the training matrix into the product of two low-rank matrices.
+    /// </summary>
+    public sealed class MatrixFactorizationPredictionTransformer : PredictionTransformerBase<MatrixFactorizationModelParameters>, ICanSaveModel
     {
-        public const string LoaderSignature = "MaFactPredXf";
-        public string MatrixColumnIndexColumnName { get; }
-        public string MatrixRowIndexColumnName { get; }
-        public ColumnType MatrixColumnIndexColumnType { get; }
-        public ColumnType MatrixRowIndexColumnType { get; }
+        internal const string LoaderSignature = "MaFactPredXf";
+        internal string MatrixColumnIndexColumnName { get; }
+        internal string MatrixRowIndexColumnName { get; }
+        internal ColumnType MatrixColumnIndexColumnType { get; }
+        internal ColumnType MatrixRowIndexColumnType { get; }
```

Comment:
Same remark here. Although I am not sure if it makes sense to make it available in both?